servers/drivers/time/rtc/main.c

```diff
@@ -0,0 +1,90 @@
+#include <resea/printf.h>
+#include <vprintf.h>
+#include <driver/io.h>
+#include <resea/ipc.h>
+#include <resea/malloc.h>
+#include <string.h>
+#include "rtc.h"
+
+static io_t rtc_io;
+
+int is_update_in_progress(void) {
+	io_write8(rtc_io, 0, 10);
+    char RTC_UIP_FLAG = io_read8(rtc_io, 1);
```

Comment:
Nit: Please avoid using uppercase characters for a variable. Use only lowercase ones. This line should be:

```c
    uint8_t status = io_read8(rtc_io, 1);
```

Reply:
Noted